README.md

```diff
@@ -78,10 +78,11 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!--- Feature Features ---->
+<!--- Future Features ---->
 
 ### Feature Features <a name="feature-features"></a>
-- Update the design
+- Update design
+- Drag and drop list elements
 
 <!-- GETTING STARTED -->
 
```

Comment:
- [ ] Kindly, Add a descriptive README file to your project, it will be great to add instructions on how to run your tests. This includes `npm install` , `npm run test`

Reply:
@usorfaitheloho their already there. 

<img width="501" alt="Captura de pantalla 2023-06-22 a la(s) 12 16 13" src="https://github.com/rudicarrilloypr/todo-list/assets/125106965/8f55b9b4-9673-44bc-8865-96f640e74a14">